src/patternfly/components/DualListSelector/dual-list-selector.scss

```diff
@@ -0,0 +1,178 @@
+.pf-c-dual-list-selector {
+  // Grid
+  --pf-c-dual-list-selector__header--GridArea: pane-header;
+  --pf-c-dual-list-selector__tools--GridArea: pane-tools;
+  --pf-c-dual-list-selector__status--GridArea: pane-status;
+  --pf-c-dual-list-selector__menu--GridArea: pane-menu;
+  --pf-c-dual-list-selector__controls--GridArea: controls;
+  --pf-c-dual-list-selector--m-chosen__header--GridArea: pane-header-c;
+  --pf-c-dual-list-selector--m-chosen__tools--GridArea: pane-tools-c;
+  --pf-c-dual-list-selector--m-chosen__status--GridArea: pane-status-c;
+  --pf-c-dual-list-selector--m-chosen__menu--GridArea: pane-menu-c;
+  --pf-c-dual-list-selector--GridTemplateColumns--pane--MinMax--min: #{pf-size-prem(200px)};
+  --pf-c-dual-list-selector--GridTemplateColumns--pane--MinMax--max: #{pf-size-prem(450px)};
+
+  // Header
+  --pf-c-dual-list-selector__header--MarginBottom: var(--pf-global--spacer--sm);
+
+  // Title
+  --pf-c-dual-list-selector__title-text--FontWeight: var(--pf-global--FontWeight--bold);
+
+  // Tools
+  --pf-c-dual-list-selector__tools--MarginBottom: var(--pf-global--spacer--md);
+  --pf-c-dual-list-selector__tools-filter--tools-actions--MarginLeft: var(--pf-global--spacer--sm);
+
+  // Menu
+  --pf-c-dual-list-selector__menu--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-dual-list-selector__menu--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-dual-list-selector__menu--MinHeight: #{pf-size-prem(200px)};
```

Comment:
should this always match the value of this: `--pf-c-dual-list-selector--GridTemplateColumns--pane--MinMax--min`